dmff/mbar.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import mdtraj as md
```

Comment:
mdtraj should be added in requirement.txt

Reply:
Done